client/signup/steps/about/index.jsx

```diff
@@ -587,6 +605,7 @@ export default connect(
 			includes( ownProps.steps, 'site-type' ) &&
 			includes( ownProps.steps, 'site-topic' ) &&
 			includes( ownProps.steps, 'site-information' ),
+		hasMultipleSites: get( getCurrentUser( state ), 'site_count', 1 ) > 0,
```

Comment:
Should this be `>= 1`? In testing for the Site Topic step, the back link is appearing when I first create a new account. It shouldn't do that.

Reply:
I've suggested an approach on https://github.com/Automattic/wp-calypso/pull/29952#issuecomment-452169016 which might help.

Let me know if this makes sense 👍